docs/content/connectors/filesystem.md

```diff
@@ -0,0 +1,76 @@
+# Filesystem
+
+FeatHub provides `FileSystemSource` to read data from file systems and `FileSystemSink`
+to materialize feature view to file systems.
+
+## Supported Processors and Modes
+
+- Local: Batch Scan, Batch Append
+- Flink: Batch Scan, Batch Append
```

Comment:
`Batch` and `Streaming` refer to the execution mode and Flink currently only supports streaming mode. Therefore, Flink should support Streaming Scan and Streaming Append. Note that Streaming Scan implies both bounded and unbounded.